yarn.lock

```diff
@@ -1046,6 +1046,11 @@
   resolved "https://registry.yarnpkg.com/@types/geojson/-/geojson-7946.0.8.tgz#30744afdb385e2945e22f3b033f897f76b1f12ca"
   integrity sha512-1rkryxURpr6aWP7R786/UQOkJ3PcpQiWkAXBmdWc7ryFWqN6a4xfK7BtjXvFBKO9LjQ+MWQSWxYeZX1OApnArA==
 
+"@types/inflection@^1.13.0":
```

Comment:
@ephys did you regenerate this file or were these changes done by merging?

Reply:
yarn handled the merge operation for that file but added inflection because I forgot to remove it from package.json